doc/newfeat.texi

```diff
@@ -4,117 +4,75 @@
-Users need to install the @code{lintr} package (available on CRAN) to
-use it. Customizable options include @code{ess-use-flymake},
-@code{ess-r-flymake-linters}, and @code{ess-r-flymake-lintr-cache}.
+@item @ESS{[R]}: The package development minor mode only activates in file-visiting buffers.
+Ones that inherit from @code{prog-mode} or @code{text-mode}. If you
+want to restore the old behavior and activate the package mode in all
+buffers (that have a @code{default-directory} that is part of a
+package path), set @code{ess-r-package-auto-activate} to @code{t}.
 
-@item Improved customization for faces.
-ESS now provides custom faces for (nearly) all faces used and places
-face customization options into their own group. Users can customize
-these options using @kbd{M-x customize-group RET ess-faces}.
+@item @ESS{[R]}: Supports flymake in R buffers for Emacs 26 and newer.
+Users need to install the @code{lintr} package to use it. Customizable
+options include @code{ess-use-flymake}, @code{ess-r-flymake-linters},
+and @code{ess-r-flymake-lintr-cache}.
 
 @item @ESS{[R]}: Gained support for xref in Emacs 25+.
 @xref{Xref,,, emacs, The Gnu Emacs Reference Manual}
 
-@item @ESS{[R]}: The startup screen has been cleaned up and displays
-the startup directory with an explicit @code{setwd()}.
+@item @ESS{[R]}: The startup screen is cleaner.
+It also displays the startup directory with an explicit @code{setwd()}.
 
-@item ESS now displays the language dialect in the mode-line
-So, for example, R buffers will now show ESS[R] rather than ESS[S].
-
-@item The ESS manual has been updated and revised.
+@item @ESS{[R]}: Changing the working directory is now always reflected in the process buffer.
 
-@item @ESS{[R]}: @code{Makevars} files are now automatically opened
-with @code{makefile-mode}.
+@item @ESS{[R]}: @code{Makevars} files open with @code{makefile-mode}.
 
 @item New variable @code{ess-write-to-dribble}.
 This allows users to disable the dribble (@code{*ESS*}) buffer if they wish.
 
-@item ESS now respects Emacs conventions for keybindings.
-This means that The @code{C-c [letter]} bindings have been
-removed. This affects @code{C-c h}, which was bound to
-@code{ess-eval-line-and-step-invisibly} in @code{sas-mode-local-map};
-@code{C-c f}, which was bound to @code{ess-insert-function-outline} in
-@code{ess-add-MM-keys}; and @code{C-c h}, which was bound to
-@code{ess-handy-commands} in @code{Rd-mode-map},
-@code{ess-noweb-minor-mode-map}, and @code{ess-help-mode-map}
-
-@item prettify-symbols-mode no longer breaks indentation
-This is accomplished by having the pretty symbols occupy the same
-number of characters as their non-pretty cousins.  You may customize
-the new variable @code{ess-r-prettify-symbols} to control this
-behavior.
-
-@item The @code{ess-r-args.el} library has obsoleted and will be removed in a future release.
+@item The @code{ess-r-args.el} library has obsoleted and will be removed in the next release.
```

Comment:
Can we make all the obsoleted/removed items together?

Reply:
Yes, done